images/linux/scripts/installers/haskell.sh

```diff
@@ -7,34 +7,29 @@
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/etc-environment.sh
 
-# Install Herbert V. Riedel's PPA for managing multiple version of ghc on ubuntu.
-# https://launchpad.net/~hvr/+archive/ubuntu/ghc
-apt-get install -y software-properties-common
-add-apt-repository -y ppa:hvr/ghc
-apt-get update
-
-# Get 3 latest Haskell Major.Minor versions
-allGhcVersions=$(apt-cache search "^ghc-" | grep -Po '(\d*\.){2}\d*' | sort --unique --version-sort)
-ghcMajorMinorVersions=$(echo "$allGhcVersions" | cut -d "." -f 1,2 | sort --unique --version-sort | tail -3)
-
-for version in $ghcMajorMinorVersions; do
-    # Get latest patch version for given Major.Minor one (ex. 8.6.5 for 8.6) and install it
-    exactVersion=$(echo "$allGhcVersions" | grep $version | sort --unique --version-sort | tail -1)
-    apt-get install -y ghc-$exactVersion
-    ghcInstalledVersions+=("$exactVersion")
-    defaultGHCVersion=$exactVersion
+# Any nonzero value for noninteractive installation
+export BOOTSTRAP_HASKELL_NONINTERACTIVE=1
```

Comment:
I think we should add this variable to etc/environments.
Otherwise, customers won' be able to use ghcup